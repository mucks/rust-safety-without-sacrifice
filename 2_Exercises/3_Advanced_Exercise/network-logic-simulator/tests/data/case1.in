# Graph build + validation + summary
NODE ForestCity weight=3 role=Validator
NODE Prospera weight=2 role=Validator
NODE Zuitzerland weight=1 role=Observer
LINK ForestCity Prospera latency=20 capacity=200
LINK Prospera Zuitzerland latency=30 capacity=150
LINK ForestCity Zuitzerland latency=70 capacity=50
SUMMARY
