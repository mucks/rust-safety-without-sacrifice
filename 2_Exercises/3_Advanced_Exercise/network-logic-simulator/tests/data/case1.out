NODES: 3, LINKS: 3
ROLES: validators=2 observers=1
LATENCY_MIN_MAX_MS: 20 70
CAPACITY_SUM_MBPS: 400
CONNECTED: true
